### PR TITLE
Lisa englisch

### PR DESCRIPTION
**Pull request by topialla**
I think I got all the issues fixed. Hopefully the capitalization etc is fine now :-)
Also the spelling suggestions from Seth are included.

### CODE REVIEW

File: labshare-client/src/views/Index.vue [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <i18n>
     {
     "en":{
-    "mainTitle": "We combine forces for more SARS-CoV-2 tests",
-    "subTitle": "Together we unleash the Power of Science!",
-    "Joinbutton": "Join now!",
+    "0title": "We combine forces for more SARS-CoV-2 tests",
+    "1title": "Together we unleash the cumulative Power of Science!",
```
Comment (SethGreylyn) [inline labshare-client/src/views/Index.vue:5]: "power" should be lowercase, but otherwise nothing stands out.

**Comment by Kavakuo** [comment]
These changes are in addition to this what @LisaR1 already fixed today (502dcf6...8acf1436ede37376a43829bdc36dae00d38d84dc) and I merged with #41?

**Comment by topialla** [comment]
Yes, they should be!

File: labshare-client/src/views/Index.vue (hunk 1 above)
Comment (topialla) [reply labshare-client/src/views/Index.vue:5]: I think it's more meant as a proper noun.

**Pull request merged**